CHANGELOG.md

```diff
@@ -53,6 +53,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 - MINOR The scratch data for all of the FEM-based physics used std::vector<type> for both the evaluation of the solution at the gauss point, but also to store the shape function (or their gradient, divergence, etc.) at the gauss points for the degrees of freedom. The issue with this is that it creates std::vector<std::vector<type>> data structures. These vectors of vectors are not contiguous in memory and may lead to a large number of cache misses as the degree of the polynomial of the shape function is increased. This PR fixes this by changing the data structure used to store the shape function (and their gradient, divergence, etc.) to use the deal.II Table<n_dim,type> data structure. These data structures are identical to vectors of vectors in the way they are used, but they store the information in a contiguous memory block (see https://dealii.org/current/doxygen/deal.II/classTableBase.html for more information). This change has minimal impact on the code, but should lead to much better scalability of the matrix and RHS assembly when the degree of the polynomial is increased and the matrix-based solvers are used. The improvement will be problem dependent, but it should never worsen performance. [#1726](https://github.com/chaos-polymtl/lethe/pull/1726)
 
+### [Master] - 2025-10-20
+
+### Added
+
+- MAJOR Allow residual normalization. This PR introduces a new parameter ("normalize residual") to the non-linear solver subsection. When the parameter is true, all residua is normalized by the volume of the triangulation. This is very convenient because, when activated, tolerances are independent of the mesh size. [1728](https://github.com/chaos-polymtl/lethe/pull/1728)
```

Comment:
```suggestion
- MAJOR Allow residual normalization. This PR introduces a new parameter ("normalize residual") to the non-linear solver subsection. When the parameter is true, all residuals are normalized by the volume of the triangulation. This is very convenient because, when activated, tolerances are independent of the mesh size. [1728](https://github.com/chaos-polymtl/lethe/pull/1728)
```